2.4/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM debian:jessie-backports
+FROM debian:stretch-backports
```

Updating to `stretch`, we shouldn't need `-backports` anymore:

```suggestion
FROM debian:stretch-slim
```